.github/workflows/unit-tests.yml

```diff
@@ -4,99 +4,55 @@ on: [push, pull_request]
 
 jobs:
   build:
-    runs-on: ubuntu-18.04
-    continue-on-error: ${{ matrix.experimental }} # Experimental builds may fail
-
-    defaults:
-      run:
-        shell: bash -l -e -o pipefail {0}
+    runs-on: ubuntu-latest
 
-    env:
-      WSI_CONDA_CHANNEL: "https://dnap.cog.sanger.ac.uk/npg/conda/devel/generic"
-      CONDA_TEST_ENVIRONMENT: "testenv"
-      CK_DEFAULT_TIMEOUT: 20
+    continue-on-error: ${{ matrix.experimental }} # Experimental builds may fail
 
     strategy:
       matrix:
         include:
-          - irods: "4.2.7"
-            server_image: "wsinpg/ub-16.04-irods-4.2.7:latest"
-            experimental: false
-          - irods: "4.2.7"
-            server_image: "wsinpg/ub-18.04-irods-4.2.10:latest"
-            experimental: true
-          - irods: "4.2.7"
+          # iRODS 4.2.11 clients on bionic
+          - irods: "4.2.11"
```

Comment:
4.1.11 is still the target AFAIK, although it has an unresolved issue in federation, possibly due to configuration. It's still being investigated. I dropped 4.2.7 for now because we are running out of time this year and it requires ubuntu Xenial, so would need a new client-building Docker image. I the iRODS update doesn't happen early next year, we'll can do the work and restore that build.